### PR TITLE
Upgrade C* library in Priam

### PR DESCRIPTION
**Pull request by jonaf**

### CODE REVIEW

File: priam/dependency-reduced-pom.xml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fahdsiddiqui) [inline priam/dependency-reduced-pom.xml]: Do we need to add dependency-reduced-pom.xml?

Comment (jonaf) [inline priam/dependency-reduced-pom.xml]: no, sorry, I didn't mean to add this.

**Comment by fahdsiddiqui** [comment]
Actually, there are a few things missing from this branch. Mainly `StandardTuner` and `PriamStartupAgent` needs to be updated. They are quick fixes if you want to get them in. https://github.com/bazaarvoice/Priam/compare/master...bazaarvoice:fsiddiqui-updateToC224#diff-be0107c7da1c99ef697b8e276ab2b9eaR31

**Comment by fahdsiddiqui** [comment]
My bad. I forgot this was being merged to my branch. Please disregard my earlier comment.

**Comment by fahdsiddiqui** [comment]
LGTM :+1:

**Pull request closed without merging**